Makefile

```diff
@@ -168,6 +184,7 @@ ddd: $(TARGET).elf
 gdb: $(TARGET).elf
 	$(GDB) -ex "target ext localhost:3333" -ex "mon reset halt" -ex "mon arm semihosting enable" $(TARGET).elf
 
+-include $(SHELL_D_FILES)
```

Comment:
Also ordering? And do we need to include the STM32F0_DRIVERS_D_FILES as well? If not, why not?